index.html

```diff
@@ -1381,6 +1381,47 @@
       </section>
     </section>
 
+    <section>
+      <h2>Verification</h2>
+
+      <p>
+This section describes a number of checks required to verify a claim.
+Some checks are essential for all verifiable claims, while some are applicable to only some claims.
+      </p>
+      <section>
+        <h3>Structural Validity</h3>
+        <p>
+          <ul>
+            <li>Document is valid JSON-LD.</li>
+            <li>Required properties are present. For example, for a Credential, <code>id</code>, <code>type</code>, and <code>claim</code> are required.</li>
```

Comment:
id isn't required, even for a Credential (bearer token). So, we will want to remove 'id' as a requirement.